packages/muelu/research/regionMG/src/SetupRegionHierarchy_def.hpp

```diff
@@ -779,7 +774,9 @@ void createRegionHierarchy(const int numDimensions,
   }
 
   RCP<Level> level0 = regHierarchy->GetLevel(0);
-  RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > revisedRowMap1 = level0->Get<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > >("regRowMap");
+  RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > regRowImporters = level0->Get<RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > >("rowImport");
```

Comment:
Is this one used anywhere? I didn't see it. If no, please remove. If yes, the shorthand `RCP<Import> regRowImporters = ...` should suffice.